rcgen/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 
 # Changes
 
-## Unreleased
+## Release 0.12.0 - December 16, 2023
 
-- Rename `RcgenError` to `Error` to avoid stuttering when used fully-qualified via `rcgen::`.
-- Upgrade to `ring` `v0.17`.
+- Rename `RcgenError` to `Error`. Contributed by [thomaseizinger](https://github.com/thomaseizinger).
+- Lock down the `From` impls for `Error` and make it future-proof.
+- Upgrade to `ring` `v0.17`. Contributed by [thomaseizinger](https://github.com/thomaseizinger).
+- Make dependency on `ring` optional and allow usage of `aws_lc_rs`. Contributed by [BiagioFesta](https://github.com/BiagioFesta).
```

Comment:
I think this would be most helpful if we explicitly say how to use aws_lc_rs:
```suggestion
- Make dependency on `ring` optional and allow usage of `aws-lc-rs`. Ring continues to be used as the default. You may opt-in to using `aws-lc-rs` instead by disabling default features and specifying the `aws_lc_rs` feature.  Contributed by [BiagioFesta](https://github.com/BiagioFesta).
```